### PR TITLE
feat: implement processed handwriting automarker example

### PR DESCRIPTION
**Pull request by matthewalex4**
- This PR provides an example of automarking maths single answer type questions.
- We use SymPy to parse the latex of a student's answer into a math expression and compare semantic equality with a model answer
- We can also check that an answer does not contain specific mathematical features (e.g. an integral). This is to prevent students from entering the question as their answer)

### CODE REVIEW

File: examples/maths-automarker.py
Comment (procaconsul) [inline examples/maths-automarker.py:1]: Bearing in mind that a section can be made up by multiple tasks, and that you could have several such one-line math tasks per section, which avenue do you want to follow?

(1) Assume sections with one task, and mark the task 0 or max-mark accordingly
(2) Assume sections with many tasks, in which case you need a notion of the amount of marks each task contributes to the max mark (similarly to what appens with multiple choice)

Both of these are fine, (1) potentially simpler. Depending on the choice, we can make small adjustments to the script :)

**Review by procaconsul — CHANGES_REQUESTED** [review]

File: examples/maths-automarker.py
Comment (matthewalex4) [reply examples/maths-automarker.py:1]: We have implemented approach (2) in commit a385dec

File: examples/maths-automarker.py [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+#       instructions: |
+#         Consider the function $x^2$.
+#       maximum mark: 30
+#       tasks:
+#       - instructions: |
+#           Enter the integral.
+#         type: maths single answer
+#       - instructions: |
+#           Enter the derivative.
+#         type: maths single answer
```
Comment (procaconsul) [inline examples/maths-automarker.py]:
```suggestion
#       - instructions: |
#           Enter the integral.
#         type: processed handwriting
#       - instructions: |
#           Enter the derivative.
#         type: processed handwriting
```

File: examples/maths-automarker.py [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+                    answer = self.answers.get(task_id, {}).get("answer")
+                    task_mark_and_answer = maths_single_answer_marks[task_id]
+                    answer = json.loads(self.answers.get(task_id, {}).get("answer"))["latex"]
+                    if task_mark_and_answer and answer:
+                        expr1 = parse_latex(answer)
+                        expr2 = parse_latex(task_mark_and_answer["answer"])
+
+                        feedback = "Awarded designated marks for answer."
+                        if (expr1.has(Integral)):
+                            feedback = "Solution should not contain an integral"
```
Comment (procaconsul) [inline examples/maths-automarker.py]: If one of the tasks has Integral in the expression, feedback is set to "Solution should not contain an integral". However, it looks like that gets overwritten on the following loop cycle (i.e. on the next task) with "Awarded designated marks for answer". You should hoist feedback around line 71 for it to stay through different loop cycles. You could use "Awarded designated marks for answer" as default feedback and then overwrite if necessary in the "Integral" condition.

**Review by procaconsul — CHANGES_REQUESTED** [review]

File: examples/maths-automarker.py (hunk 1 above)
Comment (matthewalex4) [reply examples/maths-automarker.py]: Addressed in f51a0cd

**Pull request merged**